hi-ml-azure/src/health_azure/logging.py

```diff
@@ -29,14 +29,14 @@ def logging_to_stdout(log_level: Union[int, str] = logging.INFO) -> None:
     # logging lines.
     global logging_stdout_handler
     if not logging_stdout_handler:
-        print("Setting up logging to stdout.")
+        logging.info("Setting up logging to stdout.")
```

Comment:
nitpick: At this point, logging is not yet going to stdout, so nobody will see this. hence print is better here

Reply:
Good point, will add a rank zero check before because it gets printed by each rank otherwise